### PR TITLE
Update building-a-flutter-app.md

### PR DESCRIPTION
**Pull request by icarusdust**
Fixes https://github.com/codemagic-ci-cd/codemagic-docs/issues/394

### CODE REVIEW

File: content/yaml-quick-start/building-a-flutter-app.md [hunk 1]
```diff
@@ -78,6 +78,8 @@ If your app settings in Codemagic have building Android App Bundles enabled, we
 
 Please make sure to wrap the `--bundle` pattern in single quotes. If `--bundle` option is not specified, default glob pattern `**/*.aab` will be used.
```
Comment (helinanever) [inline content/yaml-quick-start/building-a-flutter-app.md:79]:
```suggestion
Please make sure to wrap the `--bundle` pattern in single quotes. If the `--bundle` option is not specified, default glob pattern `**/*.aab` will be used.
```

Comment (icarusdust) [reply content/yaml-quick-start/building-a-flutter-app.md]: changes made

**Comment by helinanever** [comment]
1. I think the whole build commands section should be reviewed. The emphasis should be on building an app bundle, which is not mentioned at all, and it may be that our commands are outdated. See the Flutter docs: https://flutter.dev/docs/deployment/android#building-the-app-for-release
2. Can you also review the getting started flow and adding a new app part. New Flutter apps come with the Flutter workflow editor view by default, users should switch to yaml in app settings.
3. Please also review the part about exporting the configuration, the UI has changed.
4. As to the limitations between yaml vs workflow editor, in yaml, the build is stopped by default if a test script fails. Maybe we should just remove any reference to the Stop build if tests fail parameter.

File: content/yaml-quick-start/building-a-flutter-app.md [hunk 2]
```diff
@@ -76,7 +76,9 @@ If your app settings in Codemagic have building Android App Bundles enabled, we
   --key-pass $FCI_KEY_PASSWORD
 ```
 
-Please make sure to wrap the `--bundle` pattern in single quotes. If `--bundle` option is not specified, default glob pattern `**/*.aab` will be used.
+Please make sure to wrap the `--bundle` pattern in single quotes. If the `--bundle` option is not specified, default glob pattern `**/*.aab` will be used.
+
+Generally, in order to build `.aab`, command `./gradlew bundleRelease` is used and for `.apk` creation `./gradlew assembleRelease` is used. However, according to the recent updates by Google Play, only Android App Bundles are required and it replaces `.apk` as the standard publishing format.
```
Comment (helinanever) [inline content/yaml-quick-start/building-a-flutter-app.md:81]:
```suggestion
Generally, in order to build an Android App Bundle (`.aab`), the command `./gradlew bundleRelease` is used, while for `.apk` creation `the command ./gradlew assembleRelease` is used. Note that Google Play requires new apps to be published as an Android App Bundle which replaces `.apk` as the standard publishing format.
```

**Comment by icarusdust** [comment]
N 2, 3 and 4 changes made

**Comment by icarusdust** [comment]
Android app bundle as a main section updated

File: content/yaml-quick-start/building-a-flutter-app.md [hunk 3]
```diff
@@ -57,10 +57,18 @@ Set up local properties
...
 ```
-### Building .apk
+### Building an app bundle
 
 ```yaml
-- flutter build apk --release
+- flutter build appbundle --release
+```
+
+In order to build an Android App Bundle (`.aab`), the command `./gradlew bundleRelease` is used, while for `.apk` creation `the command ./gradlew assembleRelease` is used. Note that Google Play requires new apps to be published as an Android App Bundle which replaces `.apk` as the standard publishing format.
```
Comment (helinanever) [inline content/yaml-quick-start/building-a-flutter-app.md:66]: just wondering - are these commands relevant to flutter apps? to generate an app bundle, it's enough to run the `flutter build appbundle --release` above

Comment (icarusdust) [reply content/yaml-quick-start/building-a-flutter-app.md]: indeed, these commands are not needed with flutter apps. removed them.

**Review by helinanever — APPROVED** [review]
LGTM

**Comment by helinanever** [comment]
Can you also fix the formatting of the code line examples? They should not be in a notebox but formatted as a code block.

**Review by helinanever — APPROVED** [review]
